src/parachain/loans.ts

```diff
@@ -462,13 +461,19 @@ export class DefaultLoansAPI implements LoansAPI {
         return amountPerBlock.mul(blocksPerYear);
     }
 
-    _getSubsidyReward(amount: Big): MonetaryAmount<CurrencyExt> | null {
+    async _getRewardCurrency(): Promise<CurrencyExt> {
+        const rewardCurrencyId = this.api.consts.loans.rewardAssetId;
+
+        return currencyIdToMonetaryCurrency(this.assetRegistryAPI, this, rewardCurrencyId);
+    }
+
+    _getSubsidyReward(amount: Big, rewardCurrency: CurrencyExt): MonetaryAmount<CurrencyExt> | null {
```

Comment:
Realized we don't need to pass the governance currency into the constructor as the runtime contains the reward currency already. :)